docs/ad/security-ad.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: Anomaly Detection Security
+nav_order: 10
+parent: Anomaly Detection
+has_children: false
+---
+
+# Anomaly detection security
+
+You can use the security plugin with anomaly detection to limit non-admin users to specific actions. For example, you might want some users to only be able to create and view detectors, while others to modify and delete detectors.
+
+Security for anomaly detection works the same as [security for alerting](../../alerting/security/).
+
+## Basic permissions
+
+As an admin user, you can use the security plugin to assign specific permissions to users based on which APIs they need access to. For a list of supported APIs, see [Anomaly Detection API](../api/).
+
+You can also use the following built-in roles that cover most anomaly detection use cases:
+
+- `anomaly_full_access` - All access to anomaly detection APIs
+- `anomaly_read_access` - Read-only access to anomaly detection APIs
+
+If these roles don't meet your needs, mix and match individual anomaly detection [permissions](../../security/access-control/permissions/) to suit your use case. Each action corresponds to an operation in the REST API. For example, the `cluster:admin/opendistro/ad/detector/delete` permission lets you delete detectors.
+
+Sample configuration to fine-tune user access to APIs for all or none of the detectors:
```

Comment:
So I think it's better to show the role JSON (`GET _opendistro/security/api/roles/some-role`) rather than the YAML. If people are going to duplicate/modify an existing role to suit their needs, strong preference is that they use the REST API rather than sticking it in YAML files on every node and reloading.